src/OAuth2/Encryption/Jwt.php

```diff
@@ -3,170 +3,39 @@
 namespace OAuth2\Encryption;
 
 /**
- * @link https://github.com/F21/jwt
- * @author F21
+ * Bridge file to use the firebase/php-jwt package for JWT encoding and decoding.
+ * @author Francis Chuang <francis.chuang@gmail.com>
  */
 class Jwt implements EncryptionInterface
 {
-    public function encode($payload, $key, $algo = 'HS256')
+    public function __construct()
     {
-        $header = $this->generateJwtHeader($payload, $algo);
-
-        $segments = array(
-            $this->urlSafeB64Encode(json_encode($header)),
-            $this->urlSafeB64Encode(json_encode($payload))
-        );
-
-        $signing_input = implode('.', $segments);
-
-        $signature = $this->sign($signing_input, $key, $algo);
-        $segments[] = $this->urlsafeB64Encode($signature);
-
-        return implode('.', $segments);
-    }
-
-    public function decode($jwt, $key = null, $allowedAlgorithms = true)
-    {
-        if (!strpos($jwt, '.')) {
-            return false;
-        }
-
-        $tks = explode('.', $jwt);
-
-        if (count($tks) != 3) {
-            return false;
-        }
-
-        list($headb64, $payloadb64, $cryptob64) = $tks;
-
-        if (null === ($header = json_decode($this->urlSafeB64Decode($headb64), true))) {
-            return false;
+        if (!class_exists('\JWT')) {
+            throw new \ErrorException('You do not have firebase/php-jwt package installed. Please install it to use JWT features.');
```

Comment:
error message should be `firebase/php-jwt must be installed to use this feature. You can do this by running "composer require firebase/php-jwt"`